src/app/static/src/js/components/runReport/changeLog.vue

```diff
@@ -66,11 +65,11 @@
             initialType: String
         },
         methods: {
-            handleChangeLogType: function () {
-                this.$emit("changelogType", this.changeLogTypeValue)
-            },
-            handleChangeLogMessage: function () {
-                this.$emit("changelogMessage", this.changeLogMessageValue)
+            emitChangelog() {
+                this.$emit("changelog", this.changeLogMessageValue ? {
+                    message: this.changeLogMessageValue,
+                    type: this.changeLogTypeValue
+                } : null);
```

Comment:
If change log message is not populated then change log type should not be sent to orderly server ?

Reply:
Correct. Sending a null changelog is fine - OW handles this. The problem is sending a changelog without a message. So now the component only emits either null or a full object - see PR description for more on this. I looked into OW handling this case too, but it got messy and I think that ultimately it's better not to have invalid changelog entries in `RunWorkflowMetadata` in the first place.